### ISSUE TITLE
Type error when using OPTIONAL in SPARQL query

### ISSUE THREAD

**Issue by KonradHoeffner** [mgskjaeveland/sgvizler#68]
Using a SPARQL query with the OPTIONAL keyword for a table causes the following error:

```
0.122s: initializer: loading functions:   sgvizler.js:952:17
0.127s: drawing id: sgvizler-div-missingsuperclass  sgvizler.js:952:17
0.132s: loadGVizScript: loading packages:   sgvizler.js:952:17
0.227s: loadGVizScript: packages LOADED:   sgvizler.js:952:17
0.35s: sgvizler.visualization.Table -- TypeError  sgvizler.js:952:17
```

MWE:

```
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
  <script type="text/javascript" src="https://mgskjaeveland.github.io/sgvizler/v/0.6/sgvizler.js"></script>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css">
  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
  <script>
  sgvizler
  .defaultEndpointURL("https://www.snik.eu/sparql")
  .defaultChartFunction("sgvizler.visualization.Table")
  .defaultChartWidth(1000);
  $(document).ready(
    function (){ sgvizler.containerDrawAll(); }
  );
  </script>
</head>
<body>

<div id="label" data-sgvizler-query="
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
  <script type="text/javascript" src="https://mgskjaeveland.github.io/sgvizler/v/0.6/sgvizler.js"></script>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css">
  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
  <script>
  sgvizler
  .defaultEndpointURL("https://www.snik.eu/sparql")
  .defaultChartFunction("sgvizler.visualization.Table")
  .defaultChartWidth(1000);
  $(document).ready(
    function (){ sgvizler.containerDrawAll(); }
  );
  </script>
</head>
<body>

<div id="comment"  data-sgvizler-query="
select ?class ?comment
{
?class a owl:Class.
OPTIONAL {?class rdfs:comment ?comment.}
}
">
</div>

</body>
</html>

">
</div>

</body>
</html>

```

**Comment by KonradHoeffner** [comment]
P.S.: I found a workaround, maybe it helps with implementing a fix:

    OPTIONAL {?SomeSubject some:optionalProperty ?optionalVariableTmp.}
    bind(if(bound(?optionalVariableTmp),?optionalVariableTmp,"") as ?optionalVariable)

**Comment by mgskjaeveland** [comment]
Could it be that this happens only when the first result row is missing a ?comment value? I expect this is related to the bug in https://stackoverflow.com/questions/44415513/sgvizler-create-empty-table-for-empty-result

**Comment by KonradHoeffner** [comment]
I think the HTML above is broken, try the following:

```html
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
	<script type="text/javascript" src="https://www.google.com/jsapi"></script>
  <script type="text/javascript" src="https://mgskjaeveland.github.io/sgvizler/v/0.6/sgvizler.js"></script>
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.css">
  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
  <script>
  sgvizler
  .defaultEndpointURL("https://www.snik.eu/sparql")
  .defaultChartFunction("sgvizler.visualization.Table")
  .defaultChartWidth(1000);
  $(document).ready(
    function (){ sgvizler.containerDrawAll(); }
  );
  </script>
</head>
<body>

<div id="label" data-sgvizler-query="
select ?class ?comment
{
?class a owl:Class.
OPTIONAL {?class rdfs:comment ?comment.}
}
">
</div>

</body>
</html>

```

**Comment by KonradHoeffner** [comment]
It is fixed in version 0.7!

```html
<script type="text/javascript" src="https://mgskjaeveland.github.io/sgvizler/v/0.7/sgvizler.js"></script>
```
